security-privacy-questionnaire.md

```diff
@@ -88,13 +86,16 @@ created by [Web Authentication].
 
 ## 13. How does this specification distinguish between behavior in first-party and third-party contexts?
 
-This feature inherits the behavior from [Payment Request API] and
-[Web Authentication], namely:
+This feature mostly inherits the behavior from [Web Authentication], with the
+following exceptions:
 
-- A `PaymentCredential` can only be created in a first-party secure context.
-- A `PaymentCredential` can be exercised in any context where Payment Request
-  API is allowed, i.e. a first-party secure context, or a third-party secure
-  context with [delegated permission] from the top-level context.
+- A SPC-enabled `PublicKeyCredentials` can be created in a third-party secure
+  context, **for that third-party**, if given [delegated permission] from
+  the top-level context.
```

Comment:
I know that this an accurate description of a secure and privacy preserving action, but perhaps this can rephrased to mention the "RP" keyword? For example:

> When an RP is embedded in a cross origin iframe with the [delegated permission], this RP can register an SPC-enabled `PublicKeyCredential` for itself. 

Or something like that. Wordsmithing is hard.